Makefile

```diff
@@ -74,10 +74,17 @@ test3:
 docs: venv3 
 	. ./venv3/bin/activate && make -C doc html
 
+coverage: venv3
+	./venv3/bin/coverage run --source conduce -m unittest discover -v test/ 
```

Comment:
`venv3/bin/coverage` is not installed by the venv3 rule, which is the only one you've specified as a dependency, but it is installed by the deps-venv3 rule.  So should he previous line be `coverage: deps-venv3` instead?

Reply:
or are you assuming the user has done that first?

Reply:
I think I just set the wrong dependency.